test/integration/auth/node_test.go

```diff
@@ -175,6 +156,27 @@ func TestNodeAuthorizer(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	crd, err := superuserCRDClient.ApiextensionsV1beta1().CustomResourceDefinitions().Create(&apiextensionsv1beta1.CustomResourceDefinition{
```

Comment:
can this use the CRD defined under the csi-api package?

Reply:
Yup that's the plan, once the other PR merges